docs/manage/manage-subscription/create-manage-orgs-service-providers.md

```diff
@@ -70,60 +75,47 @@ After you create a child org, you can’t delete it.
 
 1. Go to **Administration** \> **Organizations**.
 1. Click **+ Add Organization**.
-
     ![orgs-page.png](/img/subscriptions/orgs-page.png)
-
 1. The **Create New Organization** pane appears.
-
     ![create-new-org-sp.png](/img/subscriptions/sp-create-new-org.png)
-
 1. **Organization Name**. Enter a unique name for the org.
 1. **Account Owner Email**. Enter the name of the account owner.
 1. **Account Owner First Name**. Enter the account owner’s first name.
 1. **Account Owner Last Name**. Enter the account owner’s last name.
 1. **Deployment**. Select a Sumo Logic deployment from the list. 
 1. **Plan Type**. By default, “POV Trial” is selected. Leave it selected. 
 1. Click **View Allocation**.
-1. The **Baselines** popup appears, and displays the ingest limits for a POV Trial org. You cannot change the limits.
-
-    ![baselines.png](/img/subscriptions/baselines.png)
-
-1. Click **Done**.
-1. Click **Save** to save the new org.
+1. The **Baselines** popup appears, and displays the ingest limits for a POV Trial org. You cannot change the limits. <br/> <img src={useBaseUrl('img/subscriptions/pov-baselines.png')} alt="pov baselines" width="450"/> 
+2. Click **Done**.
+3. Click **Save** to save the new org.
 
 ## Upgrade a POV Trial org 
 
 1. Go to **Administration** \> **Organizations**.
 1. Click the row for the trial org you want to upgrade.
-
     ![org-to-update.png](/img/subscriptions/org-to-update.png)
-
```

Comment:
Lines 95-112ish.... steps/images should be formatted like this:

1. Click the row for the trial org you want to upgrade. <br/>  ![org-to-update.png](/img/subscriptions/org-to-update.png)